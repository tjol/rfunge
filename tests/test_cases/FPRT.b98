"PDPF"4#v(0"~*~ E9.% ~*~"5F2FY7FDaF3FYM "TRPF"4#^($$D>:#,_a,v
                >:#,_v#:I@#pf0:****::*::8+b**5aa"0x%X <-> "0<
                     >0"f%"0fgF>:#,_"kos%"0aS>:#,_a,@
        >a"PDPF on"7k,            @,k7"no FPRT"a<
