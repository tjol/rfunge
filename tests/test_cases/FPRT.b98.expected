~*~ 3.571428571E+03 ~*~
0x3FE00000 <-> 1.750000
ok
